### DISCUSSION TITLE
关于安装时需要VS2017社区版本

### DISCUSSION THREAD

**Discussion by BingZi-233** [wechatferry/wechatferry#17] [Q&A]
我记得这个东西不是有一个node预编译好的版本么，为啥用这个需要VS2017社区版本自行编译的啊？
![QQ_1733450192879](https://github.com/user-attachments/assets/924f37a0-c960-4ff8-949f-edeb4ab13b2c)

**Comment by KeJunMao** [comment]
理论不需要的，看日志是 lz4 没有你的预编译版本

**Reply by KeJunMao** [reply]
lz4 用来解wx数据库的某些数据，没法不依赖，或者你有什么好的建议吗

**Reply by BingZi-233** [reply]
目前也没啥好办法，先只能这样了。

**Reply by KeJunMao** [reply]
> 目前也没啥好办法，先只能这样了。

目前只有发送 xml 的时候用的了，如果你愿意的话，可以帮忙修改成异步导入，这样的话可以把 lz4 作为可选依赖
